shaders/buffer_device_address/slang/update_vbo.comp.slang

```diff
@@ -0,0 +1,70 @@
+/* Copyright (c) 2021-2025, Arm Limited and Contributors
+ * Copyright (c) 2025, Sascha Willems
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * Licensed under the Apache License, Version 2.0 the "License";
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+// If we mark a buffer as buffer_reference, this is treated as a pointer type.
+// A variable with the type Position is a 64-bit pointer to the data within.
+// We can freely cast between pointer types if we wish, but that is not necessary in this sample.
+// buffer_reference_align is used to let the underlying implementation know which alignment to expect.
+// The pointer can have scalar alignment, which is something the compiler cannot know unless you tell it.
+// It is best to use vector alignment when you can for optimal performance, but scalar alignment is sometimes useful.
+// With SSBOs, the API has a minimum offset alignment which guarantees a minimum level of alignment from API side.
+
+// In push constant we place a pointer to VBO pointers, spicy!
+// This way we don't need any descriptor sets, but there's nothing wrong with combining use of descriptor sets and buffer device addresses.
+// It is mostly done for convenience here.
+struct Registers
+{
+    float2 **positions;
+    // A buffer reference is 64-bit, so offset of fract_time is 8 bytes.
+    float fract_time;
+}
+
+[shader("compute")]
+[numthreads(8, 8, 1)]
+void main(uint3 globalInvocationID: SV_DispatchThreadID, uint3 workgroupID: SV_GroupID, uniform Registers registers)
+{
+    uint3 workgroupSize = WorkgroupSize();
+    uint3 workgroupCount = WorkgroupCount();
```

Comment:
With VS2022, I get `error 30015: undefined identifier 'WorkgroupCount'.` here.

Reply:
Please update your Slang compiler (see issue description).

WorkgroupCount was only recently added, and e.g. the compiler from the SDK does not have that function.